spring-cloud-dataflow-server-core/src/main/java/org/springframework/cloud/dataflow/server/controller/support/ArgumentSanitizer.java

```diff
@@ -18,13 +18,17 @@
 
 import java.util.regex.Pattern;
 
+import org.springframework.cloud.dataflow.core.dsl.ArgumentNode;
+
 /**
  * Sanitizes potentially sensitive keys for a specific command line arg.
  * @author Glenn Renfro
  */
 public class ArgumentSanitizer {
```

Comment:
Did you mean this ?
https://stackoverflow.com/questions/28300232/spring-boot-actuator-hides-property-values-in-env-endpoint